src/main/java/com/softteam/vocabuilder/controller/CategoryController.java

```diff
@@ -0,0 +1,47 @@
+package com.softteam.vocabuilder.controller;
+
+import com.softteam.vocabuilder.persistence.entity.Category;
+import com.softteam.vocabuilder.service.CategoryServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.persistence.MapsId;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/categories")
+
+public class CategoryController {
+
+    @Autowired
+    private CategoryServiceImpl categoryService;
+    @PostMapping
+    public ResponseEntity<Category> createCategory(@RequestBody Category category){
+        categoryService.create(category);
+        return new ResponseEntity<Category>(HttpStatus.CREATED);
+    }
+    @PatchMapping("/update/{id}")
+    public ResponseEntity<Category> updateCategory(@PathVariable(value = "id") Long id, @RequestBody Category category){
+        category.setId(id);
+        categoryService.update(category);
+        return new ResponseEntity<Category>(HttpStatus.OK);
+    }
+    @GetMapping("/{id}")
+    public ResponseEntity<Optional<Category>> readCategory(@PathVariable(value = "id") Long id){
```

Comment:
`readX` is not a common name, the common names are `getX` or `findX`, let's use in the project `getX`

Reply:
Done getCategory()

Reply:
it says "done" in the comment, but it's not done in the PR yet 😄 
do this comments after you do the change and push the changes, for the person is reviewing see the "done" comment and also can see the change